### ISSUE TITLE
The bot does not answer anymore, nothing is displayed

### ISSUE THREAD

**Issue by Mozoloa** [matrix-discord/mx-puppet-discord#180]
Hello !

I have a pretty odd and sudden problem, my bridge works well as **I'm able to send messages and recieve them from my existing rooms**. But earlier I tried to adminme on a room and I realised the bot conversation was only me, all bot responses were deleted for some reason, and now if I talk to him I get no answer and I can't do anything.

I don't think anything changed, I tried restarting the server and the services, rerun the ansible playbook that installed in in the first place and nothing seems to fix it. **All the messaging stuff is working but the bot just ignores me.**.. what gives ? Where could I see the logs to check if everything is alright ? the SystemCTL service seems to be fine

I've used the general Ansible Playbook to install it last week, and it worked until today.

**Comment by Jafner** [comment]
+1 
I also used the popular [Ansible playbook](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-bridge-mx-puppet-discord.md) to set up my matrix server with this earlier today. I am experiencing the same issue. The bot is not even responding to my `link bot ...` command.

**Comment by TheDiscordian** [comment]
I ran into some problems recently, not 100% sure if this will help, but this is what I did and I have no issues anymore:

* Update mx-puppet-discord to the latest version (I updated Synapse as well)
* Tell the bridge bot `resendbridgeinfo` in a DM

...and that's it! I was having odd issues like messages making it to Matrix, but not going to Discord, and random Discord users not being able to get their messages through at all. However it *seems* to be okay now.
